Draft/Classes/Draft.swift

```diff
@@ -62,10 +70,19 @@ public extension Draft where ResponseType == Data {
 
 // MARK: - DecodableDraft
 
-public protocol DecodableDraft: Draft {}
+public protocol DecodableDraft: Draft {
+    var dateDecodingStrategy: JSONDecoder.DateDecodingStrategy? { get }
+}
 
 public extension DecodableDraft where ResponseType: Decodable {
+    var dateDecodingStrategy: JSONDecoder.DateDecodingStrategy? { return nil }
+    
     func convert(data: Data) throws -> ResponseType {
+        if let decodingStrategy = dateDecodingStrategy {
+            let decoder = JSONDecoder()
+            decoder.dateDecodingStrategy = decodingStrategy
+            return try decoder.decode(ResponseType.self, from: data)
+        }
         return try JSONDecoder().decode(ResponseType.self, from: data)
```

Comment:
This can be:
```
let decoder = JSONDecoder()
if let strategy = dateDecodingStrategy {
    decoder.dateDecodingStrategy = strategy
}
return try JSONDecoder().decode(ResponseType.self, from: data)
```